code-samples/eventing/helloworld/helloworld-python/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:alpine3.7
+FROM python:alpine3.9
```

Comment:
The alpine3.9 tag seems to be [out of date](https://hub.docker.com/layers/python/library/python/alpine3.9/images/sha256-442fe94d3b444303cdbb730cb56c99144d19466262249825d5b16c54ae02a265?context=explore), I think the tag that gets updates is [`3.9-alpine`](https://hub.docker.com/layers/python/library/python/3.9-alpine/images/sha256-1f078ebd3ad2405efb6dd69944046e135db45788280e1c334d10e7bf822ea842?context=explore)

Reply:
Oh thanks! I changed to the active tag [3.9-alpine](https://hub.docker.com/layers/python/library/python/3.9-alpine/images/sha256-1f078ebd3ad2405efb6dd69944046e135db45788280e1c334d10e7bf822ea842?context=explore).